prediction_market_agent/agents/safe_guard_agent/validation_result.py

```diff
@@ -4,3 +4,20 @@
 class ValidationResult(BaseModel):
     reason: str
     ok: bool
+
+
+class ValidationResultWithName(ValidationResult):
+    name: str
```

Comment:
Added the name field, because sometimes it isn't very informative without it, see:

![image](https://github.com/user-attachments/assets/3414d391-06ab-422f-84ff-8e72a0b7922a)